Release note for CIE-5485 (#3882)
SoftwareAG/c8y-docs: content/change-logs/edge/edge-2025.0.8-CIE-5485.md

---
date: 2024-01-09
title: c8yedge tool no longer crashes when run by different users
change_type:
  - value: change-VSkj2iV9m
    label: Fix
product_area: Edge
component:
  - value: component-g7hnfbu4J
    label: Edge
build_artifact:
  - value: tc-nJH2U7g3u
    label: edge
ticket: CIE-5485
version: 2025.0.8
---

> If the **c8yedge** tool has been run by one user, then running it as another user in that environment will result in a crash. Commonly this would be when you performed installation as the `root` user via `sudo`, then later upgrade or create a package as a regular "human" user. This bug has now been fixed.

Reviewer: ```suggestion
In the past, if the **c8yedge** tool was run by one user, then running it as another user in that environment resulted in a crash. Commonly, this was the case when you performed an installation as the `root` user via `sudo`, and then later upgraded or created a package as a regular "human" user. This issue has now been fixed.
```
